models/health.go

```diff
@@ -157,3 +163,10 @@ func (ps ProxyStatus) IsSynced() bool {
 func isComponentStatusSynced(componentStatus string) bool {
 	return componentStatus == "Synced"
 }
+
+func GetHealthConfiguration(annotations map[string]string) (string, error) {
+	if kialiConf, ok := annotations["health.kiali.io/rate"]; ok {
```

Comment:
Naming is hard and opnionated, just thinking in future maintenance, to me it would be more clear "kialiConfig" -> "healthConfig".
It's a minor comment.